lib/msf/core/payload/windows/reverse_winhttps.rb

```diff
@@ -114,6 +97,23 @@ def required_space
     space
   end
 
+  #
+  # Get the SSL hash from the certificate, if required.
+  #
+  def get_ssl_cert_hash
```

Comment:
Probably makes sense to toss into either the x509 utility parser or reverse_http's payload mixin (kind of weird given SSL, but its the parent for all four of those stagers)